docs/cloud/features/single_sign_on.md

```diff
@@ -0,0 +1,151 @@
+| Client ID                 | The random ID we use to communicate with their OAuth service                                                                                                                                                     | `<random string>`                                                                       |
+| Client Secret             | The random secret we use to authentication with their OAuth service                                                                                                                                              | `<random string>`                                                                       |
+| Open ID Configuration URL | This is the URL we use to gather the rest of their OpenID Configuration. We can often find this on our own and don't need to request it from them, check with the onboarding engineer to make sure we know this. | 
+
+
+### SAML V2.0
+
+This provider uses [python3-saml](https://github.com/SAML-Toolkits/python3-saml)
+to support SAML V2.0 authentication.
+
+#### Requirements
+
+If you are using a  SAML provider we need to receive three pieces of
+information from you below:
+
+| Name        | Purpose                                            | Example                                                                                                           |
+|-------------|----------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| Entity ID   | This is the providers Entity ID                    | `https://saml.example.com/entityid`                                                                               |
+| SSO URL     | This is the URL to use for SSO                     | `https://mocksaml.com/api/saml/sso`                                                                               |
+| Certificate | The certificate of the SAML Provider in PEM format | [PEM Certificates](https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/#ftoc-heading-1) |
+
+We will provide a similar set of information below:
+
+| Name        | Purpose                               | Example                                                      |
+|-------------|---------------------------------------|--------------------------------------------------------------|
+| Entity ID   | This is our Entity ID                 | `https://cloud.tobikodata.com`                               |
+| SSO URL     | This is our HTTP-Redirect Binding URL | `https://cloud.tobikodata.com/auth/saml/callback/<provider>` |
+| Certificate | Our SAML Certificate                  | **TBD**                                                      |
+
+The SSO URL will change per provider. For example if we had a provider named
+`acme`, our URL will be `https://cloud.tobikodata.com/auth/saml/callback/acme`.
```

Comment:
As stated above, the Entity ID will as well: `https://cloud.tobikodata.com/auth/saml/metadata/<name>`